src/app/home/home.component.ts

```diff
@@ -95,8 +96,20 @@ export class HomeComponent implements OnInit {
       }
     },
     {
-      appearAt: ArtsForImpactApplicationCloseDate,
-      disappearAt: "never",
+      appearAt: 'asap',
+      disappearAt: Jan8th,
+      headerText: "One donation. Twice the impact.",
+      bodyText: 'You donate.\nWe double it.',
+      backgroundImageUrl: new URL('/assets/images/blue-texture.jpg', environment.donateGlobalUriPrefix),
+      iconColor: 'primary',
+      button: {
+        text: "Explore now",
+        href: new URL('/explore/', environment.donateGlobalUriPrefix)
+      }
+    },
+    {
+      appearAt: Feb9th,
```

Comment:
if the two new cards disappear on Feb9th then I suppose there will be space to show this one again.